tests/test_unit/test_cli/test_build_deploy_snapshots/cdf_common.yaml

```diff
@@ -10,7 +10,7 @@ Database:
 - name: ingestion
 ExtractionPipeline:
 - createdBy: unknown
-  dataSetId: 0
+  dataSetId: 316416785021717804
```

Comment:
The new mocking hashes the `dataSetExternalId` string and produces a consistent 16 digit int number.